pallets/anchors-v2/src/benchmarking.rs

```diff
@@ -21,7 +21,7 @@ use super::*;
 
 #[benchmarks(
 	where
-		T: Config<Balance = u128, Hash = H256, AnchorIdNonce = u128>,
+		T: Config<Balance = u128, Hash = H256>,
```

Comment:
This is correct, and probably this pallet is only for us, so it will not be an issue. But to explain the why:

These bounds force the user of this pallet to always choose `Balance = u128` and `Hash = H256` in their runtime if they want to perform the benchmarks. Ideally, we do not want to restrict the usage of the bounds to certain concrete types. We want to allow *any* type that can be configured by the pallet.

The solution for that is to add extra bounds to those associated types as:
```rust
T: Config,
T::Balance: Into<u128>, // Or maybe T::Balance: Default is just enough
T::Hash: Default, // And use later T::Hash::default() instead of H256::from_low_u64_be(1)

Reply:
Yeah, not blocker 👍🏻